VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.html

```diff
@@ -4,103 +4,74 @@
-            <app-participants-panel></app-participants-panel>
-        </div>
-    </div>
-    <div [ngClass]="{ 'govuk-grid-column-three-quarters': !isAdminConsultation }">
+    <div>
         <div class="hearing-video">
-            <div *ngIf="isAdminConsultation || isPrivateConsultation" class="video-background">
+            <div class="video-background">
 
-                <div class="header-bar" *ngIf="isPrivateConsultation">
-                    <div class="govuk-grid-column-three-quarters govuk-body" style='float: left;'>
-                        <span class="room-title">
+                <div class="header-bar">
+                    <div class="header-content-grid govuk-body">
+                        <div class="room-title">
                             {{getCaseNameAndNumber()}}
-                        </span>
-                    </div>
-                    <div class="govuk-grid-column-one-quarter">
-                        <ng-container>
-                            <app-private-consultation-room-controls [conferenceId]="conference.id"
-                                                                    [participant]="participant"
-                                                                    [audioOnly]="audioOnly"
-                                                                    [isPrivateConsultation]="isPrivateConsultation"
-                                                                    [outgoingStream]="outgoingStream"
-                                                                    [isSupportedBrowserForNetworkHealth]="isSupportedBrowserForNetworkHealth"
-                                                                    [showConsultationControls]="showConsultationControls"
-                                                                    (leaveConsultation)="showLeaveConsultationModal()">
-                            </app-private-consultation-room-controls>
-                        </ng-container>
+                        </div>
+                        <app-private-consultation-room-controls [conferenceId]="conference.id"
+                                                                [participant]="participant"
+                                                                [audioOnly]="audioOnly"
+                                                                [isPrivateConsultation]="isPrivateConsultation"
+                                                                [outgoingStream]="outgoingStream"
+                                                                [isSupportedBrowserForNetworkHealth]="isSupportedBrowserForNetworkHealth"
+                                                                [showConsultationControls]="showConsultationControls"
+                                                                [unreadMessageCount]="unreadMessageCount"
+                                                                (leaveConsultation)="isPrivateConsultation
```

Comment:
I would think it were cleaner to have the method for the `(leaveConsultation)` to be determined in the component than in the HTML

Reply:
im not sure im following

Reply:
fixed